tabs/pid_tuning.html

```diff
@@ -43,6 +43,12 @@
                 <div class="default_btn resetbt">
                     <a href="#" id="resetProfile" i18n="pidTuningResetProfile"></a>
                 </div>
+                <div class="default_btn copyrateprofilebtn">
```

Comment:
Alternatively, how about implementing as configurator side only functionality, and not use space in the firmware for something that is never going to be used in flight?